autofs-5.1.0 - fix typo in update_hosts_mounts()

From: Ian Kent <ikent@redhat.com>

Correct the stutter in the update exports message.
---
 CHANGELOG              |    1 +
 modules/lookup_hosts.c |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/CHANGELOG b/CHANGELOG
index 7c0a7bb..645efe5 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -35,6 +35,7 @@
 - update man page autofs(8) for systemd.
 - fix fix master map type check.
 - init qdn before use in get_query_dn().
+- fix typo in update_hosts_mounts().
 
 04/06/2014 autofs-5.1.0
 =======================
diff --git a/modules/lookup_hosts.c b/modules/lookup_hosts.c
index 9d689ff..407cf31 100644
--- a/modules/lookup_hosts.c
+++ b/modules/lookup_hosts.c
@@ -214,7 +214,7 @@ next:
 			goto cont;
 
 		debug(ap->logopt, MODPREFIX
-		      "attempt to update exports for exports for %s", me->key);
+		      "attempt to update exports for %s", me->key);
 
 		master_source_current_wait(ap->entry);
 		ap->entry->current = source;
